src/Interactive/Host/Interactive/Core/InteractiveHost.cs

```diff
@@ -24,7 +26,7 @@ namespace Microsoft.CodeAnalysis.Interactive
     /// <remarks>
     /// Handles spawning of the host process and communication between the local callers and the remote session.
     /// </remarks>
-    internal sealed partial class InteractiveHost : MarshalByRefObject
+    internal sealed partial class InteractiveHost
```

Comment:
note: your PR is more than just annotating :)

Reply:
A tiny bit more. Instead of annotating `InitializeLifetimeService` I noticed it's not needed, so I deleted it :)